### PR TITLE
fix: set size of icon to lock it for this component #117

### PR DESCRIPTION
**Pull request by rmenner**
# Alaska Airlines Pull Request

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [ ] My update follows the CONTRIBUTING guidelines of this project
- [ ] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Bug Fixes:
- Fix the icon size by setting a fixed width for the auro-icon component.

### CODE REVIEW

**Comment by sun-mota** [comment]
image
<auro-badge>Nonstop</auro-badge>
Are we ok with the heights are different,  with `x` vs without `x`?

File: src/style.scss [hunk 1]
```diff
@@ -64,8 +64,7 @@
 }
 
 [auro-icon] {
-  height: var(--ds-size-300, $ds-size-300);
-  width: var(--ds-size-300, $ds-size-300);
+  width: $ds-size-300;
```
Comment (jason-capsule42) [inline src/style.scss:67]: Solution can not remove the token.

```suggestion
  width: var(--ds-size-300, $ds-size-300);
```

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/style.scss (hunk 1 above)
Comment (rmenner) [reply src/style.scss]: fixed

**Review by jordanjones243 — APPROVED** [review]

**Pull request merged**